src/pudl/analysis/plant_parts_eia.py

```diff
@@ -367,7 +367,7 @@
 
 
 @asset(
-    io_manager_key="pudl_sqlite_io_manager",
+    io_manager_key="pudl_io_manager",
```

Comment:
While merging in `main` I found this one straggler reference to the old sqlite IO manager.